packages/botonic-cli/src/commands/train.ts

```diff
@@ -29,7 +29,7 @@ class Task {
   }
 
   private logTaskPluginNotInstalled(): void {
-    console.error(
+    console.log(
```

Comment:
in a different PR, please create a log_error function in some utils file which does:
msg => console.log(colors.red(...)

and maybe a log_msg with 
msg => console.log(colors.bold(...)